assign-doi-datacite/src/test/java/no/unit/nva/doi/datacite/clients/DataCiteClientSystemTest.java

```diff
@@ -58,6 +59,12 @@
 
 class DataCiteClientSystemTest extends DataciteClientTestBase {
```

Comment:
Why are there a `DataciteClientSystemTest` and a `DataciteClientTest`. What is the difference?

Reply:
`DataciteClientSystemTest` uses wiremock.

Reply:
Maybe IntegrationTest would be better name.